addons/project_purchase/models/project_task.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class Project(models.Model):
+    _inherit = "project.task"
+
+    purchase_order_count = fields.Integer('# Purchase Orders', compute='_compute_purchase_order_count', groups='purchase.group_purchase_user')
+
+    def _compute_purchase_order_count(self):
+        purchase_orders_per_task = {purchase_order.id: count
+            for purchase_order, count in self.env['purchase.order']._read_group([
+                ('task_id', 'in', self.ids),
+            ], ['task_id'], ['__count'])}
+        for task in self:
+            task.purchase_order_count = purchase_orders_per_task.get(task.id, 0)
+
+    # ----------------------------
+    #  Actions
+    # ----------------------------
+
+    @api.model
+    def action_create_purchase_order_from_task(self):
+        self.ensure_one()
+        return {
+            'type': 'ir.actions.act_window',
+            'res_model': 'purchase.order',
+            'view_mode': 'form',
+            'context': {
+                'default_company_id': self.company_id.id or self.env.company.id,
+                'default_task_id': self.id,
+                'task_id': self.id,
+                'default_partner_id': self.partner_id.id
```

Comment:
```suggestion
                'default_partner_id': self.partner_id.id,
```
Trailing comma :innocent: